docroot/modules/custom/mass_content/mass_content.install

```diff
@@ -119,3 +109,13 @@ function mass_content_update_10004() {
     \Drupal::keyValue("system.schema")->delete($module_name);
   }
 }
+
+/**
+ * Reset state variables used by search and replace media URLs drush command.
+ */
+function mass_content_update_10005() {
```

Comment:
@arthurbaghdas  if you just need execute the same code second time and don't want have code duplication - please create `_modules_name_[helper]()` method in .install and call that code in `mass_content_update_10005` and `mass_content_update_10003`.

Reply:
I've done it the way you want. But FYI there is no need for a helper function and over complicating the code, see my previous comment. Sometimes as part of a cleanup work we do remove the old code form .install, deploy and post_update files.

Reply:
Thanks.